packages/fuel-gauge/src/coverage-contract.test.ts

```diff
@@ -386,7 +386,9 @@ describe('Coverage Contract', { timeout: 15_000 }, () => {
     const INPUT = 1;
 
     // adds the three values together, but only first param value is supplied
-    const { waitForResult } = await contractInstance.functions.echo_option_three_u8(INPUT).call();
+    const { waitForResult } = await contractInstance.functions
+      .echo_option_three_u8(INPUT, undefined, undefined)
```

Comment:
Based on #2777, this change shouldn't have been necessary. Did you get compilation errors here or did you just add these two `undefined`s based on preference?

Reply:
I had this branch lying around from before this change.

```suggestion
      .echo_option_three_u8(INPUT)
```